spec/fixtures/germany_integration_test_cases.json

```diff
@@ -134,7 +134,7 @@
   {
     "convertor": "001400",
     "valid": [
-      { "bank_code": "10090603", "account_number": "60624", "converted_bank_code": "30060601" }
+      { "bank_code": "30060601", "account_number": "60624", "converted_bank_code": "30060601" }
```

Comment:
@JoeSouthan Just double checking the right thing to do here was to update this fixture.. As the spec was failing because this bank code was removed from the yaml file, and so were all other bank codes relating to this rule..

Reply:
Yeah this has been seen before, this looks ok